modules/ROOT/pages/flex-gateway-about.adoc

```diff
@@ -0,0 +1,14 @@
+= Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+Anypoint Flex Gateway is a lightweight standalone gateway that can serve as a proxy for any API, including non-Mule APIs.
+It can run on major Linux distributions, as a containerized application using a MuleSoft supported OCI image, and as a Kubernetes-native ingress controller or API gateway.
+
+Unlike the existing API Gateway, which is embedded into Mule Runtime and depends on Mule applications to work, Anypoint Flex Gateway has a small footprint that allows for installation on the same resource, cluster, and network as the upstream target APIs, delivering sub-second response times.
```

Comment:
> the existing API Gateway
Is there an official product name for this, e.g. Anypoint Mule Gateway? Check what we're calling it in the Gateways section of docs.

Reply:
According to our Term Presentation Reference, Api Gateway is not a title or product. The API Manager documentation refers to it as API gateway.